go/cmd/aggregate-crls/aggregate-crls.go

```diff
@@ -574,8 +575,11 @@ func main() {
 
 	if err != nil {
 		glog.Fatalf("Unable to load the Mozilla issuers: %s", err)
+		return
 	}
 
+	metrics.SetGauge([]string{"IssuersAge"}, float32(mozIssuers.DatasetAge().Hours()))
```

Comment:
I think most of our duration metrics are in seconds rather than hours, but I don't really know if there is an actual convention. In any case, it's most important to be consistent within CRLite, so if we already emit other duration metrics in hours, this one should be hours, too.

Reply:
OK, seconds sounds good to me. In theory this should always be very low, since it should be fresh at each run. But this would be one where if it's more than a cronjob-period old, something has gone wrong.

Will change to seconds!